estate/models/estate_property.py

```diff
@@ -0,0 +1,97 @@
+from dateutil import relativedelta
+
+from odoo import api, fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate property"
+
+    name = fields.Char(string="Name of the Property", required=True)
+    description = fields.Text(string="Description")
+    postcode = fields.Char(string="Postcode")
+    date_availability = fields.Date(
+        string="Available From",
+        copy=False,
+        default=fields.Date.today() + relativedelta.relativedelta(months=3),
+    )
+    expected_price = fields.Float(
+        string="Expected Price",
+        required=True,
+    )
+    selling_price = fields.Float(
+        string="Selling Price",
+        readonly=True,
+        copy=False,
+    )
+    bedrooms = fields.Integer(string="Number Bedrooms", default=2)
+    living_area = fields.Integer(string="Living Area (sqm)")
+    facades = fields.Integer(string="Facades")
+    garage = fields.Boolean(string="Garage")
+    garden = fields.Boolean(string="Garden")
+    garden_area = fields.Integer(string="Garden Area (sqm)")
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[
+            ("north", "North"),
+            ("south", "South"),
+            ("east", "East"),
+            ("west", "West"),
+        ],
+    )
+    active = fields.Boolean(default=True)
+    state = fields.Selection(
+        string="State",
+        selection=[
+            ("new", "New"),
+            ("received", "Offer Received"),
+            ("accepted", "Offer Accepted"),
+            ("sold", "Sold"),
+            ("cancelled", "Cancelled"),
+        ],
+        required=True,
+        copy=False,
+        default="new",
+    )
+
+    property_type_id = fields.Many2one(
+        "estate.property.type",
+        string="Property Type",
+    )
+    users_id = fields.Many2one(
+        "res.users",
+        string="Salesman",
+        default=lambda self: self.env.user,
+    )
+    partner_id = fields.Many2one("res.partner", string="Buyer")
+    tag_ids = fields.Many2many("estate.property.tag", string="Property Tags")
+    offer_ids = fields.One2many(
+        "estate.property.offer",
+        "property_id",
+        string="Offers",
+    )
+
+    total_area = fields.Integer(string="Total Area (sqm)", compute="_compute_area")
+    best_price = fields.Integer(string="Best Offer", compute="_compute_price")
+
+    @api.depends("living_area", "garden_area")
+    def _compute_area(self):
+        for record in self:
+            record.total_area = record.living_area + record.garden_area
+
+    @api.depends("offer_ids")
```

Comment:
```suggestion
    @api.depends("offer_ids.price")
```
As is the compute will not get triggered if you only edit the price of an existing offer